common-api/src/main/scala/pl/touk/nussknacker/engine/graph/expression/Expression.scala

```diff
@@ -7,11 +7,14 @@ import io.circe.generic.JsonCodec
 object Expression {
 
   object Language {
-    val Spel         = "spel"
-    val SpelTemplate = "spelTemplate"
+    val Spel                  = "spel"
+    val SpelTemplate          = "spelTemplate"
+    val TabularDataDefinition = "tabularDataDefinition"
```

Comment:
This id will be in the scenario json. Maybe we can shortcut it a bit e.g. to `tabular`?

Reply:
I had doubts about naming it like that because it's not a language. So, I decided to explicitly say it's a data definition. I'd rather leave it as it is. In the end, this length is not a big deal.